lib/Doctrine/ORM/Internal/Hydration/ObjectHydrator.php

```diff
@@ -586,7 +586,7 @@ public function onClear($eventArgs)
     {
         parent::onClear($eventArgs);
 
-        $aliases             = array_keys($this->identifierMap);
```

Comment:
This one is wrong, it's alligned with property below - maybe just remove empty line?

Reply:
Thanks, I have done It